### PR TITLE
better code

### PR DESCRIPTION
**Pull request by nxxh447**

### CODE REVIEW

**Review by jupjohn — CHANGES_REQUESTED** [review]
This is objectively not better code.

**Comment by nxxh447** [comment]
> This is objectively not better code.

run a performance test if you want.

File: bot.js [hunk 1]
```diff
@@ -1,114 +1,59 @@
...
+client.on("PRIVMSG", (msg) => console.log(`[#${msg.channelName}] ${msg.displayName}: ${msg.messageText}`));
+
+client.on("message", async (msg) => {
+	const prefixExists = String(msg.messageText).starsWith(config.prefix);
+	const stripPrefix = String(msg.messageText).subString(1);
+	const msgText = String(msg.messageText).toLowerCase();
+	const cooldown = config.defaultCooldown;
+	
+	if (msg.senderUserID === config.botID) return;
+	if (talkedRecently.has(msg.sendUserID) && !(msg.sendUserID === config.ownerID)) return; else {
```
Comment (Felanbird) [inline bot.js:25]: Your chance of calling this better code died the moment you put an `else {` at the end of a line

File: bot.js [hunk 2]
```diff
@@ -1,114 +1,59 @@
...
+const pajladaID = "11148817";
 
-function isUpperCase(str) {
-  return str === str.toUpperCase();
-};
-
-function isLowerCase(str) {
-  return str === str.toLowerCase();
-};
+const isUpperCase = (str) => { return (str === str.toUpperCase()) };
```
Comment (ALazyMeme) [inline bot.js:9]: For someone who is fairly noobish at coding, I think "condensing" code just to preserve lines makes the readability far worse for myself.

File: bot.js [hunk 3]
```diff
@@ -1,114 +1,59 @@
-`use strict`;
+"use strict";
```
Comment (ALazyMeme) [inline bot.js:1]: My current solution of using backticks favours consistency over anything else.

File: bot.js [hunk 4]
```diff
@@ -1,114 +1,59 @@
...
-});
 
-client.on(`connect`, () => {
-  client.say(config.startupChannel, `Walking!`)
+client.on("ready", () => console.log("Successfully connected to chat"));
+client.on("close", (e) => { if () console.error("[ERR]: Failed to destroy the client.\n",e) });
+client.on("PRIVMSG", (msg) => console.log(`[#${msg.channelName}] ${msg.displayName}: ${msg.messageText}`));
+
+client.on("message", async (msg) => {
+	const prefixExists = String(msg.messageText).starsWith(config.prefix);
```
Comment (ALazyMeme) [inline bot.js:19]: My comments were removed? What purpose did this serve?

File: bot.js [hunk 5]
```diff
@@ -1,114 +1,59 @@
...
-      talkedRecently.delete(msg.senderUserID);
-    }, cooldown);
-  };
-});
 
-client.on(`connect`, () => {
-  client.say(config.startupChannel, `Walking!`)
+client.on("ready", () => console.log("Successfully connected to chat"));
+client.on("close", (e) => { if () console.error("[ERR]: Failed to destroy the client.\n",e) });
+client.on("PRIVMSG", (msg) => console.log(`[#${msg.channelName}] ${msg.displayName}: ${msg.messageText}`));
```
Comment (ALazyMeme) [inline bot.js:16]: This is where it is now inconsistent, as aforementioned.

**Review by ALazyMeme — COMMENTED** [review]
While I appreciate your effort, my "scuffed" bot actually has some logic into the way I've made it. This includes personal preference with my OCD. This repo is moreso people can see my work - no matter how good/bad it is, less so for people to make PRs.

**Pull request closed without merging**